build.yaml

```diff
@@ -1435,6 +1396,46 @@ job_type:
             ]
           }
       timeout: 120
+    run_client_installation_on_OSX:
+      at: '0 6 * * *'
```

Comment:
I would suggest here to run that job at 7:30, on the basis that the refresh OSX yosemite job runs at 5:00 and its timeout is 120minutes.
This job uses that VM